modular_skyrat/modules/importedvendors/code/vendor_snacks.dm

```diff
@@ -0,0 +1,275 @@
+// Snacks and drinks for the 'snacks' tab of vendors
+
+//Yangyu Stuff
```

Comment:
```suggestion
/*
*	Yangyu Snacks
*/
```
Was recommended by Tf4 on one of my clothing PRs to make sorting-comments look like this, helps make sure they both never mess up autodoc and never get confused for code/explanation-comments